test/testenv/testenv.go

```diff
@@ -31,7 +31,7 @@ const (
 	defaultSparkImage    = "splunk/spark"
 
 	// defaultTestTimeout is the max timeout in seconds before async test failed.
-	defaultTestTimeout = 900
+	defaultTestTimeout = 1500
```

Comment:
What is the reason for increasing this timeout?

Reply:
Because of MC deployment is taking longer hence 1500 seconds to make tests more stable. Also this worst case scenario ideally we should be waiting much less time.